domain/consensus/processes/dagtraversalmanager/window.go

```diff
@@ -28,6 +28,28 @@ func (dtm *dagTraversalManager) BlockWindow(stagingArea *model.StagingArea, high
 	return window, nil
 }
 
+func (dtm *dagTraversalManager) blockWindowHeap(stagingArea *model.StagingArea,
+	highHash *externalapi.DomainHash, windowSize int) (*sizedUpBlockHeap, error) {
+	windowHeapSlice, err := dtm.windowHeapSliceStore.Get(stagingArea, highHash, windowSize)
+	isNotFoundError := database.IsNotFoundError(err)
```

Comment:
Perhaps rename `isNotFoundError ` to `sliceNotCached`. In general the name `isNotFoundError ` is confusing also in the method below, I would avoid it